makedoc.g

```diff
@@ -9,10 +9,10 @@ LoadPackage( "AutoDoc" );
 AutoDoc( rec( 
     scaffold := rec(
         ## MainPage := false, 
-        includes := [ "intro.xml",   "print.xml",    "lists.xml", 
-                      "number.xml",  "groups.xml",   "iterator.xml", 
-                      "record.xml",  "others.xml",   "obsolete.xml", 
-                      "transfer.xml" ],
+        includes := [ "intro.xml",    "print.xml",    "lists.xml",
+                      "number.xml",   "groups.xml",   "iterator.xml",
+                      "record.xml",   "download.xml", "others.xml",
+                      "obsolete.xml", "transfer.xml" ],
```

Comment:
Meta remark (also for @cdwensley): I would suggest to change this list so that each filename is on its own line. This minimizes diffs when adding/removing/renaming files, and also IMHO it's easier to understand their order this way.